api/common/lib/DB.js

```diff
@@ -0,0 +1,6 @@
+import Sequelize from 'sequelize';
```

Comment:
move lib directory to the root of the project 
api/common/lib/DB.js -> lib/db/db.js